src/main/java/com/google/neighborgood/servlets/EditTaskServlet.java

```diff
@@ -20,44 +20,25 @@
 import com.google.appengine.api.datastore.EntityNotFoundException;
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that handles the request for editing the details and rewards for a certain task. */
+/** Servlet that handles the request for editing tasks. */
 @WebServlet("/tasks/edit")
 public class EditTaskServlet extends HttpServlet {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String keyString = request.getParameter("task-id");
-    int rewardPts = getRewardingPoints(request, "reward-input");
```

Comment:
Just a heads up: this hasn't been deleted, just moved to inside an else-block below